ormconfig.json

```diff
@@ -4,7 +4,7 @@
     "host": "localhost",
     "port": 3306,
     "username": "root",
-    "password": "",
+    "password": "root",
```

Comment:
Maybe we can find a better solution for this, maybe an ormconfig.json.dist and then ignore this one? Or inject variables via some other config file, e.g. .env if possible.

Reply:
Found something, https://github.com/typeorm/typeorm/blob/master/docs/using-ormconfig.md#using-environment-variables